package.json

```diff
@@ -67,18 +69,57 @@
     "singleQuote": true
   },
   "textlint": {
+    "plugins": {
+      "@textlint/text": {
+        "extensions": [".yml"]
+      }
+    },
+    "filters": {
+      "comments": {
+        "disablingComment": "prettier-ignore-start",
+        "enablingComment": "prettier-ignore-end"
```

Comment:
We generally prettier-ignore tabbed panes just because they confuse Prettier. Maybe we should still spell-check the content, and use text-lint specific disabling comments? Just a thought, I'm ok with this for now.

Reply:
Textlint has exactly the same issue, the tabs confuse it because it does not understand that those are code blocks (while it understands \`\`\` just fine). My line of thinking now was instead of having two comments I just reuse the ones that are there already.

This btw also [explains the extra spaces](https://github.com/open-telemetry/opentelemetry.io/pull/2791#discussion_r1205297168), the plugin requires the lines to be surrounded by newlines

Reply:
Ok, thanks for the explanation.

Yeah, I think that we need a kind of tab pane that allows use of standard markdown code fences (I've take note of this (IMHO) desirable feature).